resource/examples_customType/key_no_large/srcCpp/CustomType.cxx

```diff
@@ -0,0 +1,176 @@
+/*
+ * (c) 2005-2018  Copyright, Real-Time Innovations, Inc. All rights reserved.
+ * Subject to Eclipse Public License v1.0; see LICENSE.md for details.
+ */
+#include "CustomType.h"
+
+#ifdef RTI_WIN32
+  #define snprintf sprintf_s
+#endif
+
+/*
+ * This is the source code file that contains the implementation of the API
+ * required to work with the Custom type.
+ */
+
+DDS_LongSeq long_seq;
+
+bool initialize_custom_type_data(RTI_CUSTOM_TYPE &data)
```

Comment:
These names doesn't follow the Code Conventions.

Reply:
use initialize_custom_type_data